spec/src/main/resources/META-INF/smithy/bsp/bsp.smithy

```diff
@@ -0,0 +1,571 @@
+$version: "2"
```

Comment:
This file presents the most important part of the PR. It could be split into multiple smaller files without a problem, but I think it'll be up to the existing maintainers to decide how they want to split it.